src/pages/signin/LoginForm.js

```diff
@@ -75,10 +92,11 @@ class LoginForm extends React.Component {
                     <Text style={[styles.formLabel]}>{this.props.translate('loginForm.enterYourPhoneOrEmail')}</Text>
                     <TextInput
                         style={[styles.textInput]}
-                        value={this.state.login}
+                        ref={this.login}
```

Comment:
This doesn't look right to me. Usually for a ref we are returning a `ref` to the underlying component instance, but in this case `this.login` was set up as a string in the constructor.